<?xml version="1.0"?>
<launch>

 <!-- small description about your node -->
 
 <node pkg="glider_mission_planner" type="glider_mission_planner_node" name="GliderMissionPlannerNode" respawn="false" output="screen">
	<rosparam command="load" file="$(find glider_mission_planner)/config/config_glider_mission_planner.yaml"/>
</node>

</launch>
